src/src/user/user.service.ts

```diff
@@ -753,6 +753,33 @@ export class UserService {
 			  path
 			}
 		  }
+		  attendances {
```

Comment:
attendance or attendances?

Reply:
i have created  attendances  relationship

Reply:
why do we need attendance in regular find user by id method?